src/components/ChatLog.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+import './ChatLog.css';
+
+const ChatLog = (entries) => {
+  const getChatLogJSX = (chats) => {
+    return chats.map((chat) => {
+      return (
+        <ChatEntry
+          id={chat.id}
+          sender={chat.sender}
+          body={chat.body}
+          timeStamp={chat.timeStamp}
+          liked={chat.liked}
+          key={chat.id}
+          onLiked={entries.onLiked}
+        />
+      );
+    });
+  };
+  return <div className="chat-log">{getChatLogJSX(entries.chats)}</div>
```

Comment:
I think I see where the confusion about `entries` is. `Chatlog` is supposed to be passed a prop called `entries` and those entries are the chat objects. So you'd pass `props.entities` to the `getChatLogJSX` function. This is another reason why your tests weren't passing for this component.